R/functions.R

```diff
@@ -103,8 +123,18 @@ clique_test = function(Y,Z,Z_a,Z_b,Zobs_id,minr,minc,...){
   rtest_out = list(tobs=tobs,tvals=tvals)
   decision = out_pval(rtest_out,ret_pval,alpha=0.05)
 
+  valid_clique <- apply(conditional_clique, 2, function(x) length(unique(x))==1)

```

Comment:
see below comment.  probably should change the function name too.

Reply:
I've changed the variable name "valid_clique" to "check_clique_unique".